providers.csv

```diff
@@ -1,75 +1,75 @@
-SURF,micromobility,66c43ccb-f9f9-4f70-9707-37301b9f49a8,https://www.surfingscooters.com,https://api.app.surf/mds,https://api.app.surf/gbfs/en
-Nextbike,micromobility,31e54f5a-66c4-4a4c-bb6e-236351c90312,https://www.nextbike.de,https://platform-services.tier-services.io/data-sharing/mds/,https://api.nextbike.net/maps/gbfs/
-Starship Technologies,delivery-robots,4a1524f3-8182-4b55-90c2-023c0cac84b9,https://www.starship.xyz/,https://mds-provider.starship.xyz/api/v1/mds/,
-Deutsche Bahn Connect,micromobility,dcd47d65-b290-4bd6-adf2-24578fbd9110,https://www.deutschebahnconnect.com/,https://bike-prod.msp.dbrent.net/mds/,https://apis.deutschebahn.com/db-api-marketplace/apis/shared-mobility-gbfs/2-2/de/CallABike/gbfs/
-MyWheels,micromobility,ce0320e3-0e4b-4884-b31a-45319c0768d6,https://mywheels.nl,https://mds.mywheels.nl,
-Flamingo,micromobility,a32f88c4-ad9b-4bf1-b5d8-e0262fa8a582,https://rideflamingo.com/,https://data.rideflamingo.com/mds/,https://data.rideflamingo.com/gbfs/3/manifest.json
-3KM,micromobility,fd32a231-f373-4049-9b73-14686a24ed06,https://ario.ai,https://mds.ario.ai,
-Bauer's Intelligent Transportation,passenger-services,46454118-168c-40e1-93bb-53e68e3f03bf,https://bauersit.com,,
-MV Transportation,passenger-services,e4722796-4e0e-4203-8ae9-546bdcc4aa52,https://www.missionbaytma.org/,,
-CLS Global Transportation,passenger-services,64477ac9-0c53-4031-98a4-ffe83bd94590,https://clsgt.com/,,
-Compass Transportation,passenger-services,f00500e6-55b9-4cdf-afe9-68db2822a3e2,https://www.compasstransportation.net/,,
-Corinthian International,passenger-services,0666cc3c-a346-4413-aa16-241c48281afb,http://corinthianparking.com/,,
-Hallcon,passenger-services,d6354b9d-4179-465f-b663-648a6fc054a5,https://hallcon.com/,,
-Kaiser Permanente,passenger-services,5e96f9c3-bd41-4337-a436-475fc9747621,https://kp.org/,,
-Lux Bus America,passenger-services,0697d1bb-1678-4375-b8c6-9e09e363cbfa,https://luxbusamerica.com/,,
-Lux Leasing,passenger-services,fe7a26bc-a0bb-4943-b335-cd334d4f5fab,https://luxbusamerica.com/,,
-Royal Coach Tours,passenger-services,095da27f-7bbd-45f1-b00b-eb52b244ccd4,http://royal-coach.com,,
-Storer,passenger-services,eee92c25-89f5-4e03-a4c7-b6dac883732a,https://storercoachways.com/,,
-We Drive U Inc,passenger-services,5488c6d1-baa8-4994-8763-a2c48090631a,https://wedriveu.com/,,
-Samsara,passenger-services,aa055687-5947-4921-ae1d-70d7755609ae,https://samsara.com/,,
-Tripshot,passenger-services,e154cf80-df89-4d7e-b5e7-2ee8050a6e9c,https://www.tripshot.com/,,
-Trakk,passenger-services,6a907d9d-1e3c-4cb6-991c-f9738d30337b,https://gettrakk.com/,,
-Swoop,passenger-services,08ad2b21-8115-4d18-874f-48082e8e50fb,https://www.swoopapp.com/,,
-GoSharing,micromobility,1f859479-4427-4f52-8220-05c786436314,https://www.binbin.tech/,https://complianceapi.binbinapp.com/api/mds/,https://complianceapi.binbinapp.com/api/gbfs/
\ No newline at end of file
+provider_name,mode_id,provider_id,url,mds_api_url,gbfs_api_url,gtfs_api_url
```

Comment:
There should be a note added somewhere that this URL is to the ZIP file of the GTFS Schedule feed files.

Reply:
Not using this field now